src/foam/dao/EasyDAO.js

```diff
@@ -194,6 +194,13 @@ foam.CLASS({
             .build();
         }
 
+        if ( getLifecycleAware() ) {
```

Comment:
Can we have lifecycleAware before deletedAware? and log a deprecated warning in the deletedAware block.

Reply:
Can both be enabled at the same time? If not, we should throw an error if both are set to true ..

Reply:
Done to both.